server/localize.js

```diff
@@ -35,10 +35,12 @@ module.exports = function localize(server, options) {
     let bestLanguage = bestlang(langPrefs, webmakerI18N.getSupportLanguages(), "en-US");
     let localizedUrl;
 
+    let orgUrl = req.originalUrl;
+
     if(knownLocales.indexOf(urlLocale) !== -1) {
-      localizedUrl = req.originalUrl.replace(urlLocale, bestLanguage);
+      localizedUrl = orgUrl.replace(urlLocale, bestLanguage);
     } else {
-      localizedUrl = path.join("/", bestLanguage, req.originalUrl);
+      localizedUrl = "/" + bestLanguage + "/" + (orgUrl === "/" ? "" : orgUrl);
```

Comment:
I don't understand this conditional wouldn't:
`"/" + bestLanguage + orgUrl`
do the same thing?